src/cmd/top/joblist_pane.c

```diff
@@ -174,7 +174,19 @@ void joblist_pane_draw (struct joblist_pane *joblist)
                        name_width,
                        name_width,
                        name);
-        else
+            if (joblist->top->test_exit && joblist->top->testf)
```

Comment:
Would `top->testf` ever be set without `top->test_exit`? (i.e. are both checks really necessary here?)

Reply:
ahhh i did tweak something along the way, so you're right, both checks are no longer necessary.